encryption-service/scripts/coverage.sh

```diff
@@ -73,7 +63,7 @@ done
 
 go test -count=1 -v ./tests/...
 
-while pkill -x -SIGINT main.test; do
+while pkill -x -SIGINT encryption-serv; do
```

Comment:
Good catch with the changed name and cut-off command name. 

I think we would exchange it with a full command line match to avoid killing a wrong process. 
```suggestion
while pkill -f -SIGINT encryption-service.test; do
```

Reply:
Agreed this is much better